### ISSUE TITLE
xargs has recently started misbehaving in newEtext script on several of my 20.04 machines ... replace xargs with awk

### ISSUE THREAD

**Issue by dalehenrich** [GsDevKit/GsDevKit_home#328]
This patch has worked for me:
```
rogue:_home>git diff
diff --git a/bin/newExtent b/bin/newExtent
index 4490b161..d5ea2cdd 100755
--- a/bin/newExtent
+++ b/bin/newExtent
@@ -132,10 +132,10 @@ extractFiletypeFromSnapshot () {
     "$GEMSTONE/bin/copydbf" -i "${ssFile}" >> $tempFile 2>&1
     trap - ERR
     trap 'error ${LINENO}' ERR
-    filetype=`cat $tempFile | sed -n '/File type:/p' | cut -d : -f 2 | xargs | cut -d \  -f 1`
+    filetype=`cat $tempFile | sed -n '/File type:/p' | cut -d : -f 2 | awk '{print $1;}'`
     if [ "${filetype}x" = "x" ] ; then
       # possibly 2.x copydbf
-      filetype=`cat $tempFile | sed -n '/file type:/p' | cut -d : -f 2 | xargs | cut -d \  -f 1`
+      filetype=`cat $tempFile | sed -n '/File type:/p' | cut -d : -f 2 | awk '{print $1;}'`
     fi
   fi
 }
```

**Comment by dalehenrich** [comment]
using awk is working ... need to run through entire test suite locally before merging (in process)

**Comment by dalehenrich** [comment]
fixed with PR #329
